src/node.cc

```diff
@@ -379,6 +379,10 @@ static struct {
 #endif  //  !NODE_USE_V8_PLATFORM || !HAVE_INSPECTOR
 } v8_platform;
 
+tracing::AgentWriterHandle* GetTracingAgentWriter() {
```

Comment:
Is this abstraction layer necessary? If so could we put it in a smaller scope? I'm worried that adding
https://github.com/nodejs/node/blob/cecac2badafb14dd9236913841114387a4fa4eac/src/inspector/tracing_agent.cc#L2
Will complicate the dependency graph (hence the build, etc.)

Reply:
I’m not particularly worried about that. This seems like an appropriate place for it? It’s only being added in a single file…

Reply:
Yeah, it's an appropriate place. I'm seeing difficulty with `node_internals.h`, but we can deal with it when something actually breaks.